src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.OpenSslVersion.cs

```diff
@@ -23,7 +23,7 @@ internal static Version OpenSslVersion
                     const string OpenSSL = "OpenSSL ";
 
                     // Skip OpenSSL part, and get the version string of format x.y.z
-                    if (!Version.TryParse(OpenSslVersionDescription().AsSpan().Slice(OpenSSL.Length, 5), out s_opensslVersion))
+                    if (!Version.TryParse(OpenSslVersionDescription().AsSpan().Slice(OpenSSL.Length, 5).ToString(), out s_opensslVersion))
```

Comment:
Is implicit conversion of ```Span``` to string not supported under netcoreapp2.0? If this is left as-is the configuration breaks.

Reply:
@weshaggard  Could you take another look?

Reply:
We have implicit conversion from string to Span: https://github.com/dotnet/coreclr/blob/master/src/mscorlib/src/System/String.cs#L701

But not the other way around.